tests/test_fetch_tweets.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from unittest.mock import Mock
+import os
+import sys
+
+# Add the src directory to Python path so I can avoid restructuring this project
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+SRC_DIR = os.path.join(os.path.dirname(TEST_DIR), 'src')
+sys.path.insert(0, SRC_DIR)
+
+
+@pytest.fixture
+def mock_apis():
+    entry = {
+        'id': 'rec123',
+        'fields': {
+            'Description': 'Test pub',
+            'Search': 'https://research.arcadiascience.com/pub/test-pub',
+            'Tweets': '1734567890123456789,1734567890123456790',
+            'ID': 'test_collection_id'
+        }
+    }
+
+    twitter = Mock()
+    airtable = Mock()
+
+    airtable.get_database_entries = Mock(return_value=[entry])
+    airtable.update_page = Mock()
+
+    return {'twitter': twitter, 'airtable': airtable, 'entry': entry}
+
+
+def test_tweet_collection_and_storage(mock_apis):
```

Comment:
Same comment about naming tests after the function being tested
```suggestion
def test_get_tweets_for_entry(mock_apis):
```